jpo-ode-common/src/test/java/us/dot/its/jpo/ode/inet/InetPacketFunctionalTest.java

```diff
@@ -35,14 +35,14 @@
 
 public class InetPacketFunctionalTest {
    
-   @Capturing
-   DatagramSocket capturingDatagramSocket;
-   
-   @Capturing
-   DatagramPacket capturingDatagramPacket;
-   
-   @Capturing
-   Thread capturingThread;
+//   @Capturing
```

Comment:
Remove commented code if it's no longer needed.  Comment applies to commented out code in other files.

Reply:
It is addressed.